src/test/java/org/opensearch/ad/ml/CheckpointDaoTests.java

```diff
@@ -256,6 +257,10 @@ private ThresholdedRandomCutForest createTRCF() {
             .precision(Precision.FLOAT_32)
             .randomSeed(seed)
             .boundingBoxCacheFraction(0)
+            .transformMethod(TransformMethod.NORMALIZE)
+            .alertOnce(true)
+            .autoAdjust(true)
+            .internalShinglingEnabled(false)
```

Comment:
Is this for existing models? Single Streams? Creating external shingled models will take up more memory (and more queries)

Reply:
we only enabled internal shingling for real-time HC detectors. Other detectors all use external shingling for historical reasons.

Reply:
Makes sense. Thx.